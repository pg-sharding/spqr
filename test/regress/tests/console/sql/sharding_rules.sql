CREATE SHARDING RULE shrid1 COLUMNS id FOR DISTRIBUTION ds1;

SHOW sharding_rules;

DROP SHARDING RULE ALL;

DROP DISTRIBUTION ALL CASCADE;
DROP KEY RANGE ALL;
