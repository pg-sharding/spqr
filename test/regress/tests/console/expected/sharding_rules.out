
		SPQR router admin console
	Here you can configure your routing rules
------------------------------------------------
	You can find documentation here 
https://github.com/pg-sharding/spqr/tree/master/docs

CREATE SHARDING RULE shrid1 COLUMNS id FOR DISTRIBUTION ds1;
ERROR:  sharding rules are removed from SPQR, see https://github.com/pg-sharding/spqr/blob/master/docs/Syntax.md.
SHOW sharding_rules;
ERROR:  sharding rules are removed from SPQR, see https://github.com/pg-sharding/spqr/blob/master/docs/Syntax.md.
DROP SHARDING RULE ALL;
ERROR:  sharding rules are removed from SPQR, see https://github.com/pg-sharding/spqr/blob/master/docs/Syntax.md.
DROP DISTRIBUTION ALL CASCADE;
 drop distribution 
-------------------
(0 rows)

DROP KEY RANGE ALL;
 drop key range 
----------------
(0 rows)

